Week3/homework/step2-6.js

```diff
@@ -5,12 +5,26 @@ const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 function flattenArray2d(arr) {
   // Replace this comment and the next line with your code
-  console.log(arr);
+  const flattenedArr = [];
+  for (let i = 0; i < arr.length; i++) {
+    for (let j = 0; j < arr[i].length; j++) {
+      flattenedArr.push(arr[i][j]);
+    }
+  }
+  return flattenedArr;
 }
 
 function flattenArray3d(arr) {
   // Replace this comment and the next line with your code
-  console.log(arr);
+  const flattenedArr = [];
+  for (let i = 0; i < arr.length; i++) {
+    for (let j = 0; j < arr[i].length; j++) {
+      for (let k = 0; k < arr[i][j].length; k++) {
+        flattenedArr.push(arr[i][j][k]);
+      }
+    }
+  }
+  return flattenedArr;
```

Comment:
If you like to take a step further, try to search on google on how you can implement it using `recursion`. 🙂